README.md

```diff
@@ -1 +1,12 @@
-# opentelemetry-installer
\ No newline at end of file
+# opentelemetry-installer
+
+This is a simple script to integrate OpenTelemetry in the project. The script `otel_integrator.sh` should be copied to the root of the project.
```

Comment:
Or just fix errors: 
This is a simple script to integrate OpenTelemetry into the project